Backport: #6494 (#6506)
tendermint/tendermint: CHANGELOG_PENDING.md

- Apps

>     - [Version] \#6494 `TMCoreSemVer` is not required to be set ldflags in order to set version in Tendermint.

Author: ```suggestion
    - [Version] \#6494 `TMCoreSemVer` is not required to be set as a ldflag any longer.
```
